packages/model-viewer/rollup.config.js

```diff
@@ -80,6 +87,25 @@ if (NODE_ENV !== 'development') {
         onwarn,
       },
   );
+
+  plugins = [terser()];
+
+  outputOptions.push(
+      {
+        input: './dist/model-viewer.js',
+        output: {
+          file: './dist/model-viewer.min.js',
+          sourcemap: true,
```

Comment:
I managed to get a basic source map working here, but it's not great; it just points back to the single huge model-viewer.js file. I tried building it from lib instead with all the plugins, but it would only find two sourcemap files: constants.js and template.js. Maybe it couldn't follow deeper paths? I tried a bunch of options for sourcemap, but nothing made any difference.

Reply:
I would be interested to see if simplifying the pipeline makes a difference. I recall recently integrating terser in another application and having full source maps back to `.ts` files without much extra work (although in that case, it also required an additional plug-in to get from non-bundled `.js` to `.ts`).

Does it make a difference if you remove `cleanup(...)`?

Reply:
Thanks, I just tried, but it has the same results I've been seeing; take a look. It creates a 1.9 MB model-viewer.min.js.map file, but only two source files show up:
<img width="258" alt="Screen Shot 2020-06-02 at 7 49 08 AM" src="https://user-images.githubusercontent.com/1649964/83534364-9a175180-a4a5-11ea-8ea3-3b47603befdf.png">